tasks/queue-granules/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "target": "es2021",
+    "lib": [
+      "DOM",
+      "es2021"
+    ],
+    "outDir": "dist/src"
```

Comment:
https://github.com/nasa/cumulus/pull/3441/files/7068fdb6e12b9faad537261215f8da8052b48965#diff-5a524644461453261cb8eed4a96282c50fcd293225634ffdd3ea700eae8d13a8R164-R165 

Given the proposed changes here, should we add strictNullChecks?

Reply:
That would be my suggestion. I prefer making TS as strict as possible (perhaps being a little more forgiving with existing code to avoid a flood of compilation errors, until the code can be incrementally updated) so that the compiler catches as many errors as possible, thus reducing potential runtime errors, particularly in the case of null handling, since "null pointer" exceptions are perhaps the most common type of runtime error.